pkg/plugins/manager/client/client.go

```diff
@@ -131,6 +135,10 @@ func (s *Service) CallResource(ctx context.Context, req *backend.CallResourceReq
 	})
 
 	if err != nil {
+		if errors.Is(err, context.Canceled) {
+			return plugins.ErrPluginRequestCanceledErrorBase.Errorf("client: call resource request cancelled: %w", err)
```

Comment:
Nit for consistency with variable name:
```suggestion
			return plugins.ErrPluginRequestCanceledErrorBase.Errorf("client: call resource request canceled: %w", err)
```